x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/autoscaling/MlAutoscalingResourceTracker.java

```diff
@@ -85,6 +93,11 @@ public static void getMlAutoscalingStats(
                 .roundDown()
             : 0;
 
+        MlDummyAutoscalingEntity mlDummyAutoscalingEntity = new MlDummyAutoscalingEntity(
+            DUMMY_ENTITY_MEMORY.get(settings).getBytes(),
```

Comment:
```suggestion
            // Treat a ByteSizeValue of -1 as 0, since 0 is the default dummy entity size
            Math.max(0L, DUMMY_ENTITY_MEMORY.get(settings).getBytes()),
```